shared/monitor/monitor.py

```diff
@@ -67,7 +67,8 @@ def get_results():
     if 'kafka-sub' in j:
       sub = j['kafka-sub']
       kafka_msg = ' This data is also being published to EventStreams (kafka). Subscribe with: <p style="font-family:monospace;">' + sub + '</p>\n'
-    context = {'s': s, 'n': n, 'c': c, 'ct': ct, 'it': it, 'u': u, 'kafka_msg': kafka_msg }
+    result_locals = locals()
```

Comment:
Can we more directly use the locals dictionary rather than rebuilding a dictionary around a subset of the keys?